src/auth/auth.ts

```diff
@@ -130,10 +130,10 @@ export const signOut = (cause: SignOutCause = 'unspecified'): void => {
 };
 
 export const signIn = async (includeBillingScope = false): Promise<OidcUser> => {
-  // Here, we update `isInitialSignIn` to true, so that we can do things when the user first signs in.
+  // Here, we update `userJustSignedIn` to true, so that we can do things when the user first signs in.
```

Comment:
Thanks, this name is clearer. As a nit, I would remove "first" from the phrase "user first signs in", because this is all about the fact that the session is starting from the user clicking "Sign In", not that they are signing in to Terra for the first time.

Reply:
Updated the comment!